styles/colors-gallium-day.less

```diff
@@ -296,4 +298,4 @@
 // ---------------------------------------
 @agate-tooltip-bg-color:               @agate-darker-gray;
 @agate-tooltip-text-color:             @agate-white;
-@agate-tooltip-border-color:           @agate-dark-gray;
+@agate-tooltip-border-color:           @agate-component-text-color;
```

Comment:
Is @agate-component-text-color appropriate? It seems that @agate-component-text-color is white.

Reply:
hmm, you are right. I wonder how the ui tests passed